BLSQ/iaso
# WC2-117: Beneficiary details key information

Key information column was not needed anymore on beneficiary details submissions.

## Self proof reading checklist


## Changes

- removed unused code
- removed obsolete column


## How to test

Open a beneficiary detail page with submissions, check that the key information column is not present anymore on submissions list.

## Print screen / video

<img width="1650" alt="Screenshot 2022-11-24 at 13 19 54" src="https://user-images.githubusercontent.com/12494624/203783450-6d75ffd7-f621-4e19-a4e9-742ed00b11f8.png">
